skale/contracts/constants_holder.py

```diff
@@ -20,14 +20,15 @@
 from skale.contracts import BaseContract, transaction_method
 from skale.transactions.tools import post_transaction
 from skale.utils.constants import GAS
+from skale.dataclasses.tx_res import TxRes
 
 
-class Constants(BaseContract):
+class ConstantsHolder(BaseContract):
     @transaction_method
     def set_periods(self, new_reward_period, new_delta_period):
         op = self.contract.functions.setPeriods(new_reward_period, new_delta_period)
-        tx = post_transaction(self.skale.wallet, op, GAS['set_periods'])
-        return {'tx': tx}
+        tx_hash = post_transaction(self.skale.wallet, op, GAS['set_periods'])
```

Comment:
Not sure, but may be it's better to make post_transaction return TxRes instance

Reply:
make sense